chain/consensus/hierarchical/actors/sca/sca_cross.go

```
@@ -234,9 +236,10 @@ func incrementNonce(rt runtime.Runtime, nonceCounter *uint64) {
 func (st *SCAState) aggChildMsgMeta(rt runtime.Runtime, ch *schema.Checkpoint, aux map[string][]schema.CrossMsgMeta) {
 	for to, mm := range aux {
 		// Get the cid of MsgMeta from this subnet (if any)
-		metaIndex, msgMeta := ch.CrossMsgMeta(st.NetworkName, address.SubnetID(to))
+		sto, _ := address.SubnetIDFromString(to)
```

Why error is ignored here?